WEB/Src/PerformanceCollector/PerformanceCollector/Perf.csproj

```diff
@@ -81,8 +81,19 @@
     <AdditionalFiles Include="$(PublicApiRoot)\$(AssemblyName).dll\$(TargetFramework)\PublicAPI.Unshipped.txt" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="Implementation\QuickPulse\QuickPulseDataAccumulator.cs">
+      <ExcludeFromStyleCop>True</ExcludeFromStyleCop>
+    </Compile>
+    <Compile Update="QuickPulseTelemetryModule.cs">
+      <ExcludeFromStyleCop>True</ExcludeFromStyleCop>
+    </Compile>
+    <Compile Update="QuickPulseTelemetryProcessor.cs">
+      <ExcludeFromStyleCop>True</ExcludeFromStyleCop>
```

Comment:
I would prefer that we not Exclude any classes from our Analyzers.
If you remove this item group, how many failures do we have?
If it's small we should just fix them

Reply:
@TimothyMothra , I didn't receive any error. just warnings. Just solved all :)